### ISSUE TITLE
Paper Figure 1 numerical values for convolution

### ISSUE THREAD

**Issue by FabianHoerst** [DIAGNijmegen/StreamingCNN#8]
Hello, could you please explain how you derived the numerical values in Fig.1 in the corresponding Paper?
If I perform convolution (cross-correlation to be more precise) without padding, I get 
[-0.84, 0.05, 0.01, -0.08, 0.23, 0.07, -0.11, 0.22, 0.19, -0.55, 0.17, 1.19]
Is there an error in the paper?

**Comment by hanspinckaers** [comment]
Hi Fabian,

I added the notebook I used to make the figure: https://github.com/DIAGNijmegen/StreamingCNN/blob/master/Figure%201.ipynb

Hopefully that helps!

**Comment by FabianHoerst** [comment]
Thanks for your fast reply.

I was a little bit confused about your results since I tried to replicate it with `conv1d` operation from `torch.nn.functional` instead of using the layer `torch.nn.Conv1d`. The big difference here is the bias unit in the Conv1d layer which is set to be True for default. Thus, the results of the convolution from the paper in figure 1 are of course not concretely comprehensible, since this bias was not specified. Nevertheless, thank you for the quick feedback!

**Comment by hanspinckaers** [comment]
Well spotted! I should have removed the bias or reported it.
